docs/general/manage-tokens/how-to-transfer-rose-into-paratime.mdx

```diff
@@ -4,22 +4,24 @@ import {findSidebarItem} from '@site/src/sidebarUtils';
 # How to Transfer ROSE into a ParaTime
 
 This guide will walk you through transferring ROSE tokens from an Oasis
-consensus layer account into an Emerald, Sapphire or Cipher ParaTime using the
+consensus layer account into a Sapphire, Emerald or Cipher ParaTime using the
```

Comment:
I see we use "into" as opposed to "to" or "on to" which I slightly prefer (purely pedantically).🤔